### PR TITLE
Use expression body properties and implicit arrays where possible

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class DynamicHub : Hub
 
         public new DynamicHubClients Clients
         {
-            get { return _clients ?? new DynamicHubClients(base.Clients); }
+            get => _clients ?? new DynamicHubClients(base.Clients);
```
Comment (JamesNK) [inline src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs]: Should this return a new `DynamicHubClients` every time?

File: src/Common/ForceAsyncAwaiter.cs [hunk 2]
```diff
@@ -34,7 +34,8 @@ internal struct ForceAsyncAwaiter : ICriticalNotifyCompletion
 
         public ForceAsyncAwaiter GetAwaiter() { return this; }
 
-        public bool IsCompleted { get { return false; } } // the purpose of this type is to always force a continuation
+        public bool IsCompleted // the purpose of this type is to always force a continuation
```
Comment (davidfowl) [inline src/Common/ForceAsyncAwaiter.cs]: Fix this formatting so the comment is above the property and the => false is on the same line

File: src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs (hunk 1 above)
Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs]: No I think it should look like this https://github.com/aspnet/SignalR/blob/5b6e81387aa2a301149d9c827273379ee617d2cd/src/Microsoft.AspNetCore.SignalR.Core/Hub%60T.cs#L10-L20

Comment (JamesNK) [reply src/Microsoft.AspNetCore.SignalR.Core/DynamicHub.cs]: Ok, thought so.

**Pull request merged**